### DISCUSSION TITLE
What is CORTX

### DISCUSSION THREAD

**Discussion by zacharywhitley** [Seagate/cortx#376] [General] [answered]
I read the README and the FAQ and only came away with a vague idea that CORTX is some sort of super scalable object sore with maybe an S3 interface. It's obvious that there's been some serious work done here but after some digging around this page seems to really answer the question "what is cortx?" https://github.com/Seagate/cortx-motr/blob/main/doc/motr-in-prose.md

**Comment by johnbent** [comment, marked as answer]
Thanks @zacharywhitley , good for you for digging around and finding that!  Yes, for people, clearly like yourself, who like deep technical information, I agree with you that the [motr-in-prose.md](https://github.com/Seagate/cortx-motr/blob/main/doc/motr-in-prose.md) is currently our best document.  I'll create an issue that we should add this to our FAQ to make it more obvious so that the next person doesn't have to do the same amount of digging that you did.

ANSWER: https://github.com/Seagate/cortx-motr/blob/main/doc/motr-in-prose.md is probably currently the best document to really help answer the question about 'What is CORTX?'.
